src/js/language_label.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+var langField = document.createElement("div");
+langField.id = "language-field";
+
+/**
+ * Label for displaying the current language being used
+ */
+class LanguageControl {
+  onAdd(map) {
+    this._map = map;
+    this._container = langField;
+    this._container.style.textAlign = "right";
+    this._container.style.paddingRight = "5px";
+    this._container.style.paddingLeft = "5px";
+    this._container.style.color = "#444";
+    this._container.style.backgroundColor = "#ffffff80";
+    this._container.style.float = "right";
+    this._container.style.display = "table";
+    this._container.textContent = "";
+    return this._container;
+  }
+
+  onRemove() {
+    this._container.parentNode.removeChild(this._container);
+    this._map = undefined;
+  }
+}
+
+export var label = new LanguageControl();
+
+export function displayLocales(locales) {
+  let languageNames = new Intl.DisplayNames(locales, { type: "language" });
+  let listFormat = new Intl.ListFormat(locales, { type: "disjunction" });
+  document.getElementById("language-field").textContent = listFormat.format(
+    locales.map((locale) => languageNames.of(locale))
```

Comment:
If you want to link each language to its Wikidata item:

```js
`https://www.wikidata.org/wiki/Special:ItemByTitle/enwiki/ISO%20639:${languageCode}`
```

or to its English Wikipedia article:

```js
`https://en.wikipedia.org/wiki/Special:Search/ISO%20639:${languageCode}`
```

or to its article in the user’s preferred language:

```js
`https://hub.toolforge.org/P305:${locale}`
```

Reply:
Tracked in #611, because I'm not sure which would be the best reference.